bin/oneoff/nces_data/import_school_stats_by_years_2021_2022_pss

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env ruby
```

Comment:
A lot of this file is based on the previous private school import script: https://github.com/code-dot-org/code-dot-org/blob/staging/bin/oneoff/nces_data/import_school_stats_by_years_2019_2020_pss